packages/core/components/Modal/Modal.tsx

```diff
@@ -261,6 +269,7 @@ const ModalRenderer = (
                   key={name}
                   onMouseDown={() => setMenuOpened(name)}
                   active={active}
+                  data-testid="menu-item"
```

Comment:
Adding this `data-testid` attribute will ship it with `@react95/core` package, meaning that if someone uses this component somewhere in their application and has a test for it, they could not use this `menu-item` to test your own application. Also, despite the unity test, there is no need to have this property set on the React95 bundle prod.

Don't you agree?

Reply:
Yep, make sense. 

What you think about change the assertion to: 

```javascript
expect(getByText('Menu Text')).toBeInTheDocument();
```

This way we can remove `data-testid` attribute.

Reply:
Done [here](https://github.com/React95/React95/pull/184/commits/13bfefbce8c2dcb41003e2aa1542357f3ffbea42).